Doc/library/dis.rst

```diff
@@ -63,6 +63,27 @@ the following command can be used to display the disassembly of
 
 (The "2" is a line number).
 
+Command-line interface
+----------------------
+
+The :mod:`dis` module can be invoked as a script from the command line.
+It is as simple as:
```

Comment:
Avoid saying things are "simple": https://www.knowledgeowl.com/blog/posts/dont-say-simply-jim-fisher/

```suggestion
The :mod:`dis` module can be invoked as a script from the command line:
```

Reply:
@hugovk I agree, but `trace`, `ast` and `tokenize` docs use this wording as well :)